docs/tables/slack_conversation_members.md

```diff
@@ -0,0 +1,30 @@
+# Table: slack_conversation_members
+
+Retrieve members of a conversation.
+
+## Examples
+
+### Get member IDs in the #general channel
+
+```sql
+select
+  channel,
+  id
+from
+  slack_conversation_members
+where
+  channel IN (select id from slack_conversation where is_general);
+```
+
+### Get the information of members in the #general channel
```

Comment:
I think a better query, based on the description, would be to take each user ID we get from the #general channel, and then to join that with the `slack_user` table, and pull out columns like `display_name`, `email`, `is_admin`, `is_bot`, and `is_restricted`.